src/ShareModal.scss

```diff
@@ -0,0 +1,56 @@
+@import '_variables';
+
+.ShareModal__content {
+    padding: 2rem;
+    border-bottom: 1px solid #E4E7EB;
+}
+
+.ShareModal__form {
+    display: flex;
+    grid-template-columns: 1fr min-content;    
+    column-gap: 1rem;
+}
+
+.ShareModal__form__URL__container {
+    border: 1px solid #9DA4AF;
+    border-radius: 0.2rem;
+    max-width: 50vh;
+    padding: 0.5rem;
+}
+
+.ShareModal__form__URL {
+    overflow-x: hidden;
+    white-space: nowrap;
+}
+
+.ShareModal__form__copyButton {
+    margin: auto;
+
+    background-color: #33717F;
+    border: 0;
+    border-radius: 0.2rem;
+    color: #FFFFFF;
+    font-weight: 400;
+    padding: 0.5rem 2rem;
+    width: 10rem;
+
+    display:flex;
```

Comment:
The share button doesn't look centered, and I don't think we need some rules specified for this button. From Figma design, I see that the share button is the same width and height as footer buttons, and I think we can talk to Simon about what we talked, about making the footer button class more general and apply the style to this share button. If Simon thinks that's too out of scope, here's some style I tried and works well:

```
.ShareModal__form__copyButton {
    background-color: #33717F;
    border: 0;
    border-radius: 0.2rem;
    color: #FFFFFF;
    font-weight: 400;
    padding: 0.5rem 2rem;
    align-items: center;

    &:active {
        background-color: #ED972C;
    }

    &:focus {
        box-shadow:
            0 0 0 0.1rem #D5D8DD,
            0 0 0 $c2lc-focus-indicator-width black;
    }

    &:hover {
        background-color: #F5C58A;
        color: #1E1E1E;
    }
}
```

Reply:
I've used your styles for now.